docs/README.md

```diff
@@ -8,7 +8,32 @@ ms.topic: overview #Required
 ms.date: #Required; mm/dd/yyyy format.
 ---
 
-# Contoso rentals
+# Enterprise-grade Reference Architecture for JavaScript
+
+
+This repository contains the reference architecture and components for building enterprise-grade modern composable frontends (or micro-frontends) and cloud-native applications. It is a collection of best practices, architecture patterns, and functional components that can be used to build and deploy modern JavaScript applications to Azure.
+
+[Additional details about what the Contoso Real Estate app is and what it does from a high-level – possibly has some images from the running app].
+
+### Supported App Scenarios
```

Comment:
This is really nice to have for devs first viewing the repo. I see 8 scenarios in the diagram and scenarios docs (https://30daysof.github.io/contoso-real-estate/intro/) though but only 5 here. Do we want this overview to match up with the scenario docs?

Reply:
FYI, scenario 7 (Redis Caching) has been deprioritized and hasn't been implemented yet.